app/web_framework/cluster_base_handler.py

```diff
@@ -11,6 +12,12 @@
 from app.util.session_id import SessionId
 from app.web_framework.api_version_handler import APIVersionHandler
 
+pagination_constants = {
+    'DEFAULT_OFFSET': 0,
+    'DEFAULT_LIMIT': 20,
+    'MAX_LIMIT': 200,
+}
```

Comment:
I'd personally probably vote config, mostly because there are already established testing patterns/framework around making sure config gets reset for every test. (For example see `Configuration['timings_directory']`.)